src/components/Global/BaseSnackbar.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <v-snackbar
+    v-model="showSnackBar"
+    :color="color"
+    class="base-snackbar"
+    position="absolute"
+    location="bottom"
+    data-cy="auth-snackbar"
+    v-bind="$attrs"
+  >
+    <slot />
+    <slot name="actions">
+      <v-btn data-cy="close-snackbar" icon variant="text" @click="clear">
+        <v-icon icon="mdi-close" />
+      </v-btn>
+    </slot>
```

Comment:
There's still a bug on mobile where this shows up stacked wrong, but not part of this initial fix imo.

Reply:
![246E215E-6474-4797-B4CF-E714C6078172](https://user-images.githubusercontent.com/414475/216796231-ce6aa05c-cc96-41a8-b5ab-f373bc1ee789.png)

Reply:
Nice. I think the fix to the remaining style bug is to wrap the a lot in a template that fills the v-snackbar slot with the same name (actions).

This defines an actions slot in the base component, but it never fills the actions slot of the v-snackbar